src/components/Dapp.re

```diff
@@ -267,6 +298,14 @@ module GorillaInfo = {
         <small>
           <strong>
             <S> {Gorilla.getName(gorilla) ++ "'s Patronage: "} </S>
+            <Rimble.Tooltip
+              message={
+                "This is the total contribution that has been raised for the gorilla, "
+                ++ Gorilla.getName(gorilla)
```

Comment:
This is the total contribution that has been raised for the gorilla, Simon. (example). This sort of suggests maybe a 1 to 1 mapping? Maybe we should say This is the total contribution that has been raised thanks to the wildcard, Simon? Little bit more open ended